sdk/docs/core/README.md

```diff
@@ -106,10 +106,11 @@ Log classifications allow your application to select which specific log messages
       printf("%.*s\n", az_span_size(message), az_span_ptr(message));
    }
 
+   az_log_classification const log_classifications[] = { AZ_LOG_HTTP_REQUEST, AZ_LOG_HTTP_RESPONSE, AZ_LOG_END_OF_LIST };
```

Comment:
Is this allowed in C, will it compile? Should it be static?

Reply:
Add a comment above this line why we are declaring it here and that it can't be a local.

Reply:
This is allowed in C - it is going to be a global variable, but static won't hurt and is even better (local to a translation unit). Let me fix this.